File: src/musikdroid/app/src/main/java/io/casey/musikcube/remote/playback/Metadata.java
package io.casey.musikcube.remote.playback;

public class Metadata {
    public interface Track {
        String ID = "id";
        String EXTERNAL_ID = "external_id";
        String URI = "uri";
        String TITLE = "title";
        String ALBUM = "album";
        String ALBUM_ID = "album_id";
        String ALBUM_ARTIST = "album_artist";
        String ALBUM_ARTIST_ID = "album_artist_id";
        String GENRE = "genre";
        String TRACK_NUM = "track_num";
        String GENRE_ID = "visual_genre_id";
        String ARTIST = "artist";
        String ARTIST_ID = "visual_artist_id";
    }

    public interface Album {
        String TITLE = "title";
        String ALBUM_ARTIST = "album_artist";
    }

    private Metadata() {

    }
}

File: src/musikdroid/app/src/main/java/io/casey/musikcube/remote/playback/Metadata.kt
package io.casey.musikcube.remote.playback

object Metadata {
    interface Track {
        companion object {
            val ID = "id"
            val EXTERNAL_ID = "external_id"
            val URI = "uri"
            val TITLE = "title"
            val ALBUM = "album"
            val ALBUM_ID = "album_id"
            val ALBUM_ARTIST = "album_artist"
            val ALBUM_ARTIST_ID = "album_artist_id"
            val GENRE = "genre"
            val TRACK_NUM = "track_num"
            val GENRE_ID = "visual_genre_id"
            val ARTIST = "artist"
            val ARTIST_ID = "visual_artist_id"
        }
    }

    interface Album {
        companion object {
            val TITLE = "title"
            val ALBUM_ARTIST = "album_artist"
        }
    }
}
